pkg/tools/gen/genopenapi.go

```diff
@@ -375,10 +377,15 @@ func GetPkgDir(base string, pkgName string) string {
 }
 
 // GetKclOpenAPIType converts the kcl.KclType(the representation of Type in KCL API) to KclOpenAPIType(the representation of Type in KCL Open API)
-func GetKclOpenAPIType(pkgPath string, from *kcl.KclType, nested bool) *KclOpenAPIType {
+func GetKclOpenAPIType(pkgPath string, from *kcl.KclType, nested bool, base_schema *gpyrpc.KclType) *KclOpenAPIType {
```

Comment:
I think the parameter `base_schema` is unnecessary here, because base_schema is existed in `from.BaseSchema`